docs/pages/components/shellbar.md

```diff
@@ -56,9 +56,9 @@ This example shows the minimum shellbar for a single application product with on
     <div class="fd-shellbar__action">
       <div class="fd-popover fd-popover--right">
         <div class="fd-popover__control">
-          <div class="fd-button fd-shellbar__button fd-user-menu__control" aria-controls="WV3AY276" aria-expanded="false" aria-haspopup="true" role="button">
+          <button class="fd-button fd-shellbar__button fd-shellbar__button--user-menu" aria-controls="WV3AY276" aria-expanded="false" aria-haspopup="true" role="button">
```

Comment:
Now that these have been changed to `button`, you can go ahead and remove `role="button"` for all of these.